satellite/marketing/marketingweb/server.go

```diff
@@ -186,11 +190,23 @@ func formToStruct(w http.ResponseWriter, req *http.Request) (o marketing.NewOffe
 	return o,nil
 }
 
+func (s *Server) reRenderTables(w http.ResponseWriter){
```

Comment:
Don't use shorthands, use properly named things. `re` usually means `regular expression`.

Reply:
i think faris just meant "rerender" as one word?

Reply:
I think it might mean `redirect`.

Reply:
It's redirecting to "/" which re-renders to table so both. Maybe I'll go with `refreshTables` ?

Reply:
There's no need for the separate func, a `http.Redirect(w, r, "/", http.StatusSeeOther)` should suffice.